aries-site/src/data/structure.js

```diff
@@ -49,10 +50,10 @@ export const structure = [
   },
   {
     name: 'Learn',
-    color: 'purple!',
+    color: 'blue!',
     description:
       'Learn foundational knowledge and best practices for how to build HPE applications with Grommet using these tutorials, how-to guides, and explanations.',
-    icon: (size, color) => <IconSquare size={size} color={color} />,
+    icon: color => <Grow color={color} />,
```

Comment:
For some reason I am not seeing the color come through:
![Screenshot 2023-01-11 at 1 57 42 PM](https://user-images.githubusercontent.com/1756948/211916346-da4332c3-c9f0-4e2b-903f-22c38bc16b5f.png)

stroke and fill aren't in the CSS

Reply:
Strange, let me look into that it was working on my local deploy.

Reply:
Im not seeing the `icon` is that also missing?